src/redux/reducers.js

```diff
@@ -0,0 +1,23 @@
+const initialState = {
+  countryName: 'Am',
+  countryData: [],
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case 'SET_COUNTRY_NAME':
+      return {
+        ...state,
+        countryName: action.payload,
+      };
+    case 'SET_COUNTRY_DATA':
+      return {
+        ...state,
+        countryData: action.payload,
+      };
+    default:
+      return state;
+  }
+};
+
+export default reducer;
```

Comment:
**OPTIONAL** - You have done well so far, but I suggest that you use the redux toolkit for your redux store, the createSlice and createAsyncThunk hooks will help you avoid a lot of boilerplate code that comes with using redux. Please fix this 🙏